### ISSUE TITLE
Newly added emotes not showing up

### ISSUE THREAD

**Issue by RandomChatterino** [SevenTV/chatterino7#242]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

When a new emote gets added it doesn't show up on Chatterino7 compared to the 7TV extension.
(yes even with a refresh of emotes, reconnect, turning off and on, reinstalling etc)

As can be seen in the screenshot, the emote pentawTF showing up on the browser with 7TV but not Chatterino7.
Stream: twitch.tv/penta

### Screenshots

![2023-10-02 02_09_57-Chatterino 7 4 6](https://github.com/SevenTV/chatterino7/assets/146682403/d1113422-4306-4d9f-857f-8be5e76ffc8c)

### OS and Chatterino Version

Chatterino 7.4.6 (commit f2d677b2) built with Qt 6.5.0, MSVC 193532217 Running on Windows 10 Version 22H2, kernel: 10.0.19045

**Comment by Nerixyz** [comment]
Try [clearing your cache](https://wiki.chatterino.com/Settings/#how-do-i-delete-the-chatterino-cache-settings) and reopen Chatterino.

**Comment by RandomChatterino** [comment]
Without clearing the cache the pentawTF emote showed up after a few hours. Before making the report I had manually deleted all Chatterino7 and Chatterino2 folders. 

On the channel twitch.tv/zayiro the "KirbyDance" emote doesn't show up, that emote has been added to the channel for a **long time**. But doesn't show up on Chatterino7. I reloaded the emotes checked, closed and deleted the cache manually again, didn't fix it.

![2023-10-02 15_11_19-Chatterino 7 4 6](https://github.com/SevenTV/chatterino7/assets/146682403/13b72dd6-f26e-4e69-9075-914c6e48afe7)


Edit. I installed Chatterino7 on a separate laptop behind a VPN and the emote (KirbyDance) doesn't show there either.
